src/index.ts

```diff
@@ -82,6 +84,18 @@ program
             }
         }
 
+        if(cmd.customResourceAttributes){
```

Comment:
Need tests for this block

Reply:
Tests added in b0c9a22. Please let me know if this new style is ok otherwise I'll change it to reflect the others.